### PR TITLE
bithumb: Add missing functionality

### PR DESCRIPTION
**Pull request by shazbert**
# PR Description

Adds Missing API functionality: Assets status

This alerts caller of current withdrawal and deposit status. 

Fixes # (issue)

## Type of change

Please delete options that are not relevant and add an `x` in `[]` as item is complete.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How has this been tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration and
also consider improving test coverage whilst working on a certain feature or package.

- [ ] go test ./... -race
- [ ] golangci-lint run
- [ ] Test X

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation and regenerated documentation via the documentation tool
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally and on Github Actions/AppVeyor with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: exchanges/bithumb/bithumb_types.go [hunk 1]
```diff
@@ -290,3 +290,14 @@ type OHLCVResponse struct {
...
 	Data   [][6]interface{} `json:"data"`
 }
+
+// Status defines the current exchange allowance to deposit or withdraw a
+// currency
+type Status struct {
+	Status string `json:"status"`
+	Data   struct {
+		DepositStatus    int `json:"deposit_status"`
+		WithdrawalStatus int `json:"withdrawal_status"`
```
Comment (gloriousCode) [inline exchanges/bithumb/bithumb_types.go:300]: int64

**Review by thrasher- — APPROVED** [review]
tACK

File: exchanges/bithumb/bithumb_test.go [hunk 2]
```diff
@@ -706,3 +706,15 @@ func TestGetAmountMinimum(t *testing.T) {
 		})
 	}
 }
+
+func TestGetAssetStatus(t *testing.T) {
```
Comment (gloriousCode) [inline exchanges/bithumb/bithumb_test.go:710]: t.Parallel?

**Review by gloriousCode — CHANGES_REQUESTED** [review]
`{"status":"0000","data":{"withdrawal_status":0,"deposit_status":0}}` 🎉

**Review by gloriousCode — APPROVED** [review]
tACK! Thanks for making those changes!

**Pull request merged**